doc/source/whatsnew/v0.24.0.rst

```diff
@@ -73,6 +73,25 @@ as ``.values``).
    ser.array
    ser.to_numpy()
 
+:meth:`~Series.to_numpy` gives some control over the ``dtype`` of the resulting :class:`ndarray`,
```

Comment:
would rather you put this in the docs themselves (basics?) and just keep this note shorter

Reply:
Sure.